prov/sm2/src/sm2.h

```diff
@@ -123,6 +129,36 @@ struct sm2_xfer_entry {
 	uint8_t user_data[SM2_INJECT_SIZE];
 } __attribute__((packed));
 
+struct sm2_atomic_secondary_hdr {
+	// What atomic operation is being done
+	uint8_t datatype;
+	uint8_t atomic_op;
+
+	// Where the data is going on the receiver side
+	int rma_ioc_count;
+	struct fi_rma_ioc rma_ioc[SM2_IOV_LIMIT];
+
+	// This is needed for writing the result, all data will be kept in the
```

Comment:
Match the description of field format that you use in the xfer_hdr.

Reply:
deleted comments